sdk/src/dlob/DLOBSubscriber.ts

```diff
@@ -25,13 +31,24 @@ export class DLOBSubscriber {
 	intervalId?: NodeJS.Timeout;
 	dlob = new DLOB();
 	public eventEmitter: StrictEventEmitter<EventEmitter, DLOBSubscriberEvents>;
+	env : DriftEnv;
```

Comment:
can just get this off the DriftClient. but if we remove phoenix/serum from here, idt you'll need it here